packages/hebao_v1/contracts/base/BaseModule.sol

```diff
@@ -155,4 +155,28 @@ contract BaseModule is Module, ReentrancyGuard
             w.bindStaticMethod(methods[i], address(0));
         }
     }
+
+    function transact1(
```

Comment:
Instead of `transact1` and `transact2`, why not something more clear like `transactCall` and `transactDelegateCall`?

Reply:
done